package.json

```diff
@@ -19,71 +17,63 @@
     "ckeditor5",
     "ckeditor 5"
   ],
+  "type": "module",
+  "exports": {
+    ".": {
+      "types": "./dist/plugin.d.ts",
+      "import": "./dist/ckeditor.js"
+    },
+    "./package.json": "./package.json"
+  },
   "dependencies": {
     "lodash-es": "^4.17.21"
   },
   "peerDependencies": {
-    "vue": "^3.0.0",
-    "ckeditor5": ">=42.0.0 || ^0.0.0-nightly"
+    "ckeditor5": ">=42.0.0 || ^0.0.0-nightly",
+    "vue": "^3.3.0"
```

Comment:
The `defineModel()` macro is available since v3.4+. Should we change the Vue peer dependency to `^3.4.0`?